tools/ds-identify

```diff
@@ -553,6 +553,12 @@ dscheck_CloudStack() {
     return $DS_NOT_FOUND
 }
 
+dscheck_Exoscale() {
+    is_container && return ${DS_NOT_FOUND}
+    dmi_product_name_matches "Exoscale*" && return $DS_FOUND
```

Comment:
Note: this will not work right now (we have a cloudstack product name), but that won't be a problem since we instead will rely on the datasource being set in the templates, in cloud.cfg.d/

We will change our DMI product name accordingly, and so it will work in the future in the general case (once a newer cloud-init trickles down to all the templates).